events/messageCreate.js

```diff
@@ -1,13 +1,82 @@
 const { Events } = require('discord.js');
 
+const fs = require('fs');
+
+let response_common = [];
+let response_rare = [];
+let response_epic = [];
+let response_legendary = [];
+
+function fetchMessages() {
+  fs.readFile('response_common.txt', 'utf8', (err, data) => {
```

Comment:
This line is repeated 4 times in the file. Do you think it's possible to create a pass the file name to the function and return the buffer to a variable. Let me know if you need further clarification.